tox.ini

```diff
@@ -82,3 +82,15 @@ commands = pylint --rcfile=tox.ini apptemplates
 deps =
     django111: Django>=1.11,<2.0
     pylint
+
+[travis:env]
+DJANGO =
+    1.4: django14
+    1.5: django15
+    1.6: django16
+    1.7: django17
+    1.8: django18
+    1.9: django19
+    1.10: django110
+    1.11: django111
+    flake8: flake8
```

Comment:
I don't think this line is needed for flake8, if you use the matrix include as motivated above.